docs/src/main/sphinx/release/release-469.md

```diff
@@ -0,0 +1,125 @@
+# Release 469 (8 Jan 2025)
+
+## General
+
+* Refrain from join pushdown for modified tables. ({issue}`24447`)
+* Fix parsing of negative `0x`, `0b`, `0o` long literals. ({issue}`24601`)
+
+## Security
+
+## Web UI
+
+## JDBC driver
+
+* Add `physicalInputTimeMillis` to `io.trino.jdbc.QueryStats`. ({issue}`24604`)
+* Fix loading spooled segment data when cluster is secured. ({issue}`24595`)
+
+## Server RPM
+
+## Docker image
+
+## CLI
+
+* Fix loading spooled segment data when cluster is secured. ({issue}`24595`)
+
+## BigQuery connector
+
+## Blackhole connector
+
+## Cassandra connector
+
+## ClickHouse connector
+
+## Delta Lake connector
+
+* Improve performance of reading from new Delta Lake tables by compressing files
+  with `ZSTD` by default, instead of the previous `SNAPPY`.  ({issue}`17426`)
+* Improve performance of queries on tables with large transaction log JSON
+  files. ({issue}`24491`)
+* Fix JMX metrics for file system caching with multiple catalogs. ({issue}`24510`)
+
+## Druid connector
+
+## Elasticsearch connector
+
+## Exasol connector
+
+## Faker connector
+
+* Add support for views. ({issue}`24242`)
+* Support generating sequences. ({issue}`24590`)
+* {{breaking}} Replace predicate pushdown with `min`, `max`, and `options`
+  column properties. ({issue}`24147`)
+
+## Google Sheets connector
+
+## Hive connector
+
+* Fix JMX metrics for file system caching with multiple catalogs. ({issue}`24510`)
+* Fix reading from tables when using the Alluxio file system. ({issue}`23815`)
```

Comment:
@raunaqmorarka is this really only for the Hive connector .. or also Iceberg, Delta Lake, Hudi?

Reply:
Iceberg and delta as well

Reply:
thx